text-editor/text-editor-003-en.md

```diff
@@ -1 +1,102 @@
 ## Insertion of a tweet : a case study
+
+In this chapter, we will try to give a better insight of how the whole thing is
+working, by breaking down the insertion of a tweet case and the components
+involved in it's workflow.
+
+### Getting the HTML markup from twitter
```

Comment:
I think it'd be nice to describe the behavior of this feature first : be able to insert a rendered tweet into the edited text, just by pasting the URL of the tweet (ie no more handmade HTML edition to insert an embed of a tweet).